internal/service/ec2/vpnsite_connection.go

```diff
@@ -189,6 +189,38 @@ func ResourceVPNConnection() *schema.Resource {
 					ValidateFunc: validation.StringInSlice(vpnTunnelOptionsIKEVersion_Values(), false),
 				},
 			},
+			"tunnel1_log_options": {
+				Type:     schema.TypeSet,
```

Comment:
```suggestion
				Type:     schema.TypeList,
				MaxItems:     1,
```

as there is at most a single `VpnTunnelLogOptionsSpecification`.

Reply:
Fixed :+1: